eng/Version.Details.xml

```diff
@@ -299,9 +299,9 @@
          aren't shipping, or those extensions packages depend on aspnetcore packages that won't ship. However, given the cost
          of maintaining this coherency path is high. This being toolset means that aspire is responsible for its own coherency.
     -->
-    <Dependency Name="Microsoft.NET.Sdk.Aspire.Manifest-9.0.100-preview.1" Version="9.0.0-preview.2.24163.9">
+    <Dependency Name="Microsoft.NET.Sdk.Aspire.Manifest-9.0.100-preview.1" Version="9.0.0-preview.3.24175.2">
       <Uri>https://github.com/dotnet/aspire</Uri>
-      <Sha>9faf59f870abdeb427c51c1380fce84d8163f2f0</Sha>
+      <Sha>05244f8670d138019d2283cdc8be4052e4990bc3</Sha>
     </Dependency>
     <!-- Intermediate is necessary for source build. -->
     <Dependency Name="Microsoft.SourceBuild.Intermediate.aspire" Version="9.0.0-preview.2.24163.9">
```

Comment:
The version of the intermediate also needs to be updated, will change

Reply:
Why doesn't it get auto-updated? cc @MichaelSimons

Reply:
It didn't get updated because for some reason the build didn't publish the intermediate.  This is a question for the aspire team to investigate.  @eerhardt - Do you help direct to who could help investigate?

Reply:
cc @joperezr @RussKie

Reply:
Which repo is this package getting built from? I can't see this package being built in dotnet/aspire...

Reply:
Darc states that `dotnet/aspire` is the repository building the intermediate, but it doesn't seem to have any `preview3` versions for it

Reply:
Yeah, we should be producing this package. Looks like with the move to 1ES we are no longer building source-build, meaning we are not producing source build assets. @RussKie can you please take a look at this?

Reply:
Thanks for the nudge. There was an infra pile which went unnoticed... Long story short there's now a 9.0.0-preview.4.xxx package on the dotnet9-transport feed available for the consumption.
Apologies for the delay.